messaging/kafka/mux/multiplexer.go

```diff
@@ -216,9 +216,12 @@ func (mux *Multiplexer) genericConsumer() {
 		case msg := <-mux.consumer.Config.RecvMessageChan:
 			mux.Debug("Kafka message received")
 			mux.propagateMessage(msg)
-			// Mark offset as read. If the Multiplexer is restarted it
-			// continues to receive message after the last committed offset.
-			mux.consumer.MarkOffset(msg, "")
+			// Mark offset for hash/random partitioners
+			if mux.partitioner != client.Manual {
```

Comment:
We might want to do one formal thing. Create github issue that would represent the feature request and add refference to code that this line fixes it.